src/search.cpp

```diff
@@ -1271,11 +1272,13 @@ namespace {
               rm.selDepth = thisThread->selDepth;
               rm.scoreLowerbound = rm.scoreUpperbound = false;
 
-              if (value >= beta) {
+              if (value >= beta)
+              {
                  rm.scoreLowerbound = true;
                  rm.uciScore = beta;
               }
-              else if (value <= alpha) {
+              else if (value <= alpha)
+              {
                  rm.scoreUpperbound = true;
                  rm.uciScore = alpha;
```

Comment:
If you like, you can fix the indentation here as well (3 spaces → 4 spaces).

Reply:
Thanks, I've included this and some other changes.